app/styles/object_inspector.scss

```diff
@@ -10,3 +10,107 @@
   margin-left: 39px;
   word-break: break-all;
 }
+
+:root {
+  --pink: #909;
+  --green: #318409;
+  --black: #000;
+  --light-black: #333;
+  --white: #fff;
+  --light-pink: #b96aa0;
+}
+
+.icon-service {
+  background-color: #76c76b;
+  border-color: #569c69 !important;
+  &::before {
+    content: "S";
+  }
+}
+
+.icon-computed {
+  background-color: #e358b1;
+  border-color: #740674  !important;
+  &::before {
+    content: "C";
+  }
+}
+
+.mixin__property--group {
+  font-weight: bold;
+}
+
+.mixin__property-dependency-list, .mixin__property-dependency-item {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: block;
+  margin-bottom: 0.2em;
+}
+
+.mixin__property-dependency-item {
+  margin-left: 2.5em;
+  position: relative;
+}
+
+.mixin__property-dependency-item--computed {
+  color: var(--light-pink);
+}
+
+.mixin__property-name--computed {
+  color: var(--pink);
+}
+.mixin__property-name--service {
+  color: var(--green);
+}
+
+.mixin__property-dependency-item::before {
+  content: "";
+  display: block;
+  background: url("data:image/svg+xml;utf8,<svg width='8' height='16' xmlns='http://www.w3.org/2000/svg'><g transform='translate(-14 -16)' stroke='#979797' fill='none' fill-rule='evenodd'><circle cx='18' cy='28' r='3'/><path d='M18 24.571V16.5'/></g></svg>") no-repeat;
+  width: 8px;
+  height: 16px;
+  position: absolute;
+  bottom: 0.2em;
+  left: -0.8em;
+}
+
+.mixin__property-dependency-item:first-child::before {
+  background: url('data:image/svg+xml;utf8,<svg width="17" height="15" version="1" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd" stroke="#979797"><circle cx="13" cy="11" r="3"/><path d="M13 8c0-3-1-4-4-4H5C2 4 1 3 1 0"/></g></svg>') no-repeat;
```

Comment:
Let's try
background: url('data:image/svg+xml;utf8,<svg width="17" height="15" xmlns="http://www.w3.org/2000/svg"><g transform="translate(1)" stroke="#979797" fill="none" fill-rule="evenodd"><circle fill="#FFF" cx="12" cy="11" r="3"/><path d="M12 8c0-3-1-4-4-4H4C1 4 0 3 0 0"/></g></svg>') no-repeat;

Reply:
`data:image/svg+xml;utf8,<svg` doesn't seem to working for me in Firefox